### PR TITLE
video prompt stage0.1

### PR DESCRIPTION
**Pull request by benitogf**
https://github.com/mozilla/chatspaces/issues/28

wow I'm such a doge :) all day to just 5 lines

The next button is currently doing the send, something tell me that it should do something else, but I need more brieffing on this

### CODE REVIEW

File: public/javascripts/controllers.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ angular.module('chatspace.controllers', []).
       $location.path('/dashboard');
     };
 
-    $rootScope.toggleSettings = function () {
+    $rootScope.toggleSettings = function () {
```
Comment (fwenzel) [inline public/javascripts/controllers.js]: no

File: routes/index.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ module.exports = function (app, io, nconf, parallax, usernamesDb, crypto, Parall
             });
           });
         } catch (e) {
-          console.err(e);
+          //console.err(e);
```
Comment (fwenzel) [inline routes/index.js:61]: Hm do we really just want to swallow errors?

Comment (operatorjen) [inline routes/index.js:61]: that may be my fault.. i can fix after this

Comment (benitogf) [inline routes/index.js:61]: I did't like it, but I wasnt able to run the app, this line was trowing an error of e not defined

File: public/javascripts/controllers.js (hunk 1 above)
Comment (benitogf) [inline public/javascripts/controllers.js]: on it, sorry

Comment (fwenzel) [inline public/javascripts/controllers.js]: no worries, I wasn't trying to be abrasive, just concise :)

**Pull request merged**